spec/features/books/book_spec.rb

```diff
@@ -0,0 +1,14 @@
+RSpec.describe 'Books page', type: :feature do
+  let!(:book) { create(:book) }
+
+  it 'Not found' do
+    visit book_path(100)
+    expect(page).to have_content "The page you were looking for doesn't exist."
+  end
+
+  it 'Click book link' do
```

Comment:
Did you read it https://docs.google.com/document/d/1ZYOsWtNNU8J3lPZf0GaXtNTQ8hPnh-acpIgRG0VakFc/edit#heading=h.17dp8vu?